.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: c
+sudo: required
+dist: trusty
+compiler:
+  - gcc
+os: linux
+before_install:
+- sudo apt-get -qq update
+- sudo apt-get install -y dpkg-dev debhelper dkms lintian
+- sudo apt-get install linux-headers-`uname -r`
+- sudo cp -R . /usr/src/digimend-7
+script:
+- make
+- sudo dkms add -m digimend -v 7
+- sudo dkms build -m digimend -v 7
+- sudo dkms install -m digimend -v 7
+- sudo dkms uninstall -m digimend -v 7
```

Comment:
Could you somehow make this work without explicitly mentioning the version number? Either by omitting it altogether or extracting it automatically somehow. This way we won't have to update this file too on release.